plugins/module_utils/network/eos/rm_templates/snmp_server.py

```diff
@@ -0,0 +1,1227 @@
+                "location": "{{ loc }}"
+            },
+        },
+        {
+            "name": "notification",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\snotification\slog\sentry\slimit
+                \s+(?P<num>\d+)
+                *$""",
+                re.VERBOSE,
+            ),
+            "setval": "snmp-server notification log entry limit {{ notification }}",
+            "result": {
+                "notification": "{{ num }}"
+            },
+        },
+        {
+            "name": "objects.mac",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\sobjects
+                \s+mac-address-tables
+                \s+disable
+                *$""",
+                re.VERBOSE,
+            ),
+            "setval": "snmp-server objects mac-address-tables disable",
+            "compval": "objects",
+            "result": {
+                "objects": {
+                    "mac_address_tables": "{{ True }}"
+                }
+            },
+        },
+        {
+            "name": "objects.route",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\sobjects
+                \s+route-tables
+                \s+disable
+                *$""",
+                re.VERBOSE,
+            ),
+            "setval": "snmp-server objects route-tables disable",
+            "compval": "objects",
+            "result": {
+                "objects": {
+                    "route_address_tables": "{{ True }}"
+                }
+            },
+        },
+        {
+            "name": "qos",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\sqos\sdscp
+                \s+(?P<num>\d+)
+                *$""",
+                re.VERBOSE,
+            ),
+            "setval": "snmp-server qos dscp {{ qos }}",
+            "result": {
+                "qos": "{{ num }}"
+            },
+        },
+        {
+            "name": "qosmib",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\sqosmib\scounter-interval
+                \s+(?P<num>\d+)
+                *$""",
+                re.VERBOSE,
+            ),
+            "setval": "snmp-server qosmib counter-interval {{ qosmib }}",
+            "result": {
+                "qosmib": "{{ num }}"
+            },
+        },
+        {
+            "name": "transmit",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\stransmit\smax-size
+                \s+(?P<num>\d+)
+                *$""",
+                re.VERBOSE,
+            ),
+            "setval": "snmp-server transmit max-size {{ transmit }}",
+            "result": {
+                "transmit": "{{ num }}"
+            },
+        },
+        {
+            "name": "transport",
+            "getval": re.compile(
+                r"""
+                \s*snmp-server\stransport\stcp
```

Comment:
is variable missing? (?P<tcp>) something like this

Reply:
No. As of now only tcp is supported under transport.